pinc/dpsql.inc

```diff
@@ -59,7 +57,8 @@ function dpsql_dump_query_result($result, $start_at = 0, $show_rank = DPSQL_NO_R
     echo "</table>\n";
 }
 
-function dpsql_dump_themed_query_result($result, $start_at = 0, $show_rank = DPSQL_NO_RANK, $ralign_ints = DPSQL_RIGHT_ALIGN_INTS)
+/** @param bool|mysqli_result $result */
+function dpsql_dump_themed_query_result($result, int $start_at = 0, bool $show_rank = DPSQL_NO_RANK): void
```

Comment:
I think this would work as we're not on PHP 8.1, which supports union types:

```suggestion
function dpsql_dump_themed_query_result(bool|mysqli_result $result, int $start_at = 0, bool $show_rank = DPSQL_NO_RANK): void
```